### DISCUSSION TITLE
"Repository job log" should be linked to from Dependency Dashboard too (not just from PRs)

### DISCUSSION THREAD

**Discussion by MPV** [renovatebot/renovate#27175] [Suggest an Idea]
### Tell us more.

All PRs created by Renovate contains this really helpful piece (for troubleshooting any issues, and understanding what it did under the hood):
>View repository job log [here](https://app.renovatebot.com/dashboard#github/denoland/deno).

But what about if/when Renovate doesn't have open PRs (or is failing for some reason)?

Could we please also have such a link from the Dependency Dashboard too?

I've had users confused by this, i.e:
>1. "How do I find what happened in Renovate"?
>2. "How do I troubleshoot why my `renovate.json` didn't give me the PRs I intended it to?"

**Comment by MPV** [comment]
Related:
- https://github.com/renovatebot/renovate/discussions/11892

**Comment by rarkins** [comment]
I think we could inject this link using either dependencyDashboardHeader or dependencyDashboardFooter. It cannot be a link to a job _log_ otherwise it would require us to update the dashboard issue every single job. Instead it can link to the repo page.

Would you suggest header? If so then it would be added to this top text: 
<img width="708" alt="image" src="https://github.com/renovatebot/renovate/assets/6311784/3893dccf-00c9-4073-aacc-7efe01491469">

**Reply by karfau** [reply]
I really like the idea, and would prefer the footer, (just because it's similar to the default PR footer).

The issue I just encountered when configuring the `dependencyDashboardFooter` as part of our shared config, it **overrides** the checkbox to trigger a rerun on the repository.

So I would say either [the docs](https://docs.renovatebot.com/configuration-options/#dependencydashboardfooter) need to be extended mentioning that using the footer will replace those things, and maybe provide guidance on how to extend instead of replace it, or these things should just be independent of the footer template variable.

Update:
```
- [ ] <!-- manual job -->Check this box to trigger a request for Renovate to run again on this repository
```
seems to be the default footer
